api/app/fireweather_bot/common.py

```diff
@@ -92,6 +94,15 @@ def get_station_names_to_codes() -> dict:
     return station_codes
 
 
+def get_station_names_to_codes_wfwx() -> dict:
+    """ Helper function to create dictionary of (station_name: station_code) key-value pairs
+        specifically from WFWX API
+    """
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+    return loop.run_until_complete(wildfire_one.get_stations())
```

Comment:
I'd prefer this function indicate that it's synchronous/blocking so that someone doesn't accidentally use it in the API. Also - it's worth re-writing this code so that the auth token is only created once, and then passed to the station list as well as the hourly observations.